common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1714,7 +1746,7 @@ object CelebornConf extends Logging {
       .stringConf
       .checkValue(
         value => Seq("reduce", "map", "mapgroup").contains(value.toLowerCase(Locale.ROOT)),
-        s"Invalid split mode, Celeborn only support partition type of (reduce, map, mapgroup)")
+        s"Invalid partition type, Celeborn only support partition type of (reduce, map, mapgroup)")
```

Comment:
we can call `.transform(_.toLowerCase)` before `checkValue(...)` so that we need not call `toLowerCase` again when use it.